common-ktx/src/test/java/co/nimblehq/common/extensions/DateRangeTest.kt

```diff
@@ -0,0 +1,15 @@
+package co.nimblehq.common.extensions
+
+import org.junit.Assert
+import org.junit.Test
+import java.util.*
+
+class DateRangeTest {
+
+    @Test
+    fun `when checking date in range, it should return the correct answer`() {
```

Comment:
How about setting the test with the `private` access modifier? If so, please apply for all the test cases for consistency.

Reply:
@hoangnguyen92dn Opps, I've updated 🙏